src/Raven.Server/Documents/Queries/Results/QueryResultRetrieverBase.cs

```diff
@@ -69,11 +120,13 @@ public abstract class QueryResultRetrieverBase : IQueryResultRetriever
         private QueryTimingsScope _functionScope;
         private QueryTimingsScope _loadScope;
```

Comment:
Please check the warning below - `The field 'QueryResultRetrieverBase<TDocument>._loadScope' is never used`